.circleci/config.yml

```diff
@@ -24,8 +24,20 @@ jobs:
           command: ./gradlew --stacktrace lint checkstyle
       - android/save-gradle-cache
       - android/save-lint-results
+  Test:
+    executor:
+      name: android/default
+      api-version: "27"
```

Comment:
nit / tip: we could also declare an `executor` at the top of the `config.yml` then point to it by name in each job (see [here for an example in their documentation](https://circleci.com/docs/2.0/configuration-reference/#executors-requires-version-21)).

The benefit being that if we change the `api-version` or executor image in the future we'll only have to change it in a single place. nbd but thought it could be a nice addition if you wanted to take advantage of it

Reply:
TIL! This seems nice to have!

Reply:
Thanks for the tip @AliSoftware! In one of our private repos, I had used a Markdown anchor, but this key is much better. I'll update before merging! 👍

Reply:
Yeah I love YAML anchors too! And basically CircleCI provide a couple of mechanisms to declare stuff ahead of time (like `executor`) that could already be achieved just by using the feature built in YAML for anchors and references… but I guess they added the dedicated keys like `executor` mainly because a lot of people don't know about YAML anchors and thus this was easier to discover for most poeple?

Reply:
Yeah. Native key is definitely better.

I looked into this, and it turns out that with the `executor: name: android/default` syntax we are already referencing a preconfigured executor: the one [defined in our orb](https://github.com/wordpress-mobile/circleci-orbs/blob/72e11900f381d2389c9339a3def26b65b029f5bf/src/android/orb.yml#L6-L13).

I tried, but there is no way I could find to use/redefine our `android/default` as one of the custom executors defined with the `executors` key.

Your suggestion regarding centralizing the point of change still stands, though, and it's something I'm keen to get sorted.

There are at least two options:

- Use an anchor
- Drop the `api-version` parameter and rely on the default value defined in the orb (we reference the orb by version, so the value won't change behind the scene and result in non-deterministic builds)

I feel this is a discussion for a dedicated PR, though, so I'll merge this in the meantime 😄